src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -634,9 +633,8 @@ MStatus PRTModifierAction::updateDynamicEnums() {
 					const std::wstring currString = stringArray[enumIndex];

```

Comment:
Optimization: consider turning this into a std::wstring_view and avoid the string copy

Reply:
Using wstring_view now.
However, in case we need the substring, I still need to create a new string, because the MString constructor only allows null-terminated strings in the constructor. (and a string_view only points to a wchar and stores the stringlength but cannot not return a null-terminated string other than the source string)